docs/pages/architecture/tls-routing.mdx

```diff
@@ -25,6 +25,13 @@ To implement TLS routing, Teleport uses SNI ([Server Name Indication](https://en
 and ALPN ([Application-Level Protocol Negotiation](https://en.wikipedia.org/wiki/Application-Layer_Protocol_Negotiation))
 TLS extensions.
 
+<Admonition
```

Comment:
Since this is Admonition is rather long, I think it would make sense to turn this into an H2-level section, e.g., "## When to use TLS routing", with the same content.

Reply:
I like the idea of using a warning Admonition because it puts a big yellow box on the page and makes look at it. I'll move the content to a separate section and trim down the Admonition to hint that TLS routing may not be for everyone.